### DISCUSSION TITLE
LSP(?) proposal: insert empty comment after `Enter` when writing multi-line comments.

### DISCUSSION THREAD

**Discussion by hayleigh-dot-work** [gleam-lang/gleam#1873] [Ideas & suggestions]
Because gleam doesn't have a multi-line comment syntax (which is fine) it becomes a bit tedious having to write a bunch of `/// ` when writing multi-line docs.

I'd really love it if there was a way for my editor to just insert a blank comment after a newline if I do:
```
/// blah blah
/// blah blah<Enter>
```
so that i end up with
```
/// blah blah
/// blah blah
/// 
```

Not entirely sure what it would take to make that happen, though. Might not be LSP related at all 🙃

**Comment by lpil** [comment]
This is an editor feature rather than LSP. I used to have this in neovim, I would love it in vscode if someone knows how to do that!

**Reply by inoas** [reply]
The LSP could add the empty `///` or `////` line if not given for consistency that seems to have established itself?

```gleam
/// blah blah
/// blah blah 
```

```gleam
/// blah blah
/// blah blah
///
```

**Reply by lpil** [reply]
We could possibly have the formatter do this but generally we have said the formatter does not alter comments.
